types/react-navigation/index.d.ts

```diff
@@ -659,6 +666,31 @@ export function StackRouter(
   routeConfigs: NavigationRouteConfigMap,
   config: NavigationTabRouterConfig
 ): NavigationRouter<any, any, any>
+
+/**
+ * Create Navigator
+ *
+ * @see https://github.com/react-community/react-navigation/blob/master/src/navigators/createNavigator.js
+ */
+export function createNavigator(
+  router: NavigationRouter<any, any, any>,
+  routeConfigs?: NavigationRouteConfigMap,
+  navigatorConfig?: any,
```

Comment:
Nah its doesn't have to be, it could be Tab, Drawer, Stack. The flow types changed recently.